host_core/test/docker-compose.yml

```diff
@@ -1,9 +1,13 @@
 version: "3"
 services:
   nats:
-    image: nats:2.3.4
+    image: nats:2
```

Comment:
This threw me but apparently this is maintained as the latest 2.x version so that works for us.